.github/workflows/docs-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI Documentation
```

Comment:
We want to make sure we run these tests locally and fix the test failures: https://github.com/arnav-mandal1234/deltacode/actions/runs/4491761503/jobs/7900679423#step:6:30 This is not visible on the PR for some reason (?)

Reply:
Removed. Since this PR is not merged, failing actions are not visible. This feature comes with the skeleton upgrade.